### PR TITLE
qa,test,pybind: drop py2 support

### PR DESCRIPTION
**Pull request by tchaikov**
the goal is to ditch `six` module, and drop py2 support. 

this patchset is the first one in a serial.

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: qa/tasks/ceph_manager.py [hunk 1]
```diff
@@ -13,10 +13,9 @@
 import threading
 import traceback
 import os
-import six
 
 from io import BytesIO
```
Comment (dillaman) [inline qa/tasks/ceph_manager.py:17]: Nit: perhaps combine? `from io import BytesIO, StringIO`

Comment (tchaikov) [reply qa/tasks/ceph_manager.py:17]: updated and repushed.

**Comment by tchaikov** [comment]
jenkins test make check

**Comment by tchaikov** [comment]
changelog

- fix the regression introduced in `insights/module.py`.

**Comment by tchaikov** [comment]
jenkins test make check

**Comment by dillaman** [comment]
RBD tests seem fine

**Comment by batrick** [comment]
CephFS tests are good.

**Comment by tchaikov** [comment]
https://pulpito.ceph.com/kchai-2020-07-03_01:47:32-rados-wip-kefu-testing-2020-07-01-2343-distro-basic-smithi/

**Pull request merged**